### ISSUE TITLE
Clipping labels on horizontal self loops

### ISSUE THREAD

**Issue by bentvelj** [CSchank/finsm#98]
The label for a transition will clip inside the arrow in self loops, only when the the loop is oriented horizontally, as a C (or backwards C). Making the self transition bigger or smaller does not affect it.
![image](https://user-images.githubusercontent.com/45087499/76156943-e08c2880-60cf-11ea-914e-697553867313.png)

**Comment by Necried** [comment]
Yep, it's pretty obvious when it happens. We'll have to figure out the placements better with this.
